src/sentry/static/sentry/app/views/settings/components/tag.tsx

```diff
@@ -37,29 +42,20 @@ const Tag = styled(
   display: inline-flex;
   box-sizing: border-box;
   padding: ${p => (p.size === 'small' ? '0.1em 0.4em 0.2em' : '0.35em 0.8em 0.4em')};
-  font-size: 75%;
+  font-size: ${p => p.theme.fontSizeExtraSmall};
   line-height: 1;
-  color: ${p => (p.priority ? '#fff' : p.gray5)};
+  color: ${p => (p.priority ? '#fff' : p.theme.gray5)};
```

Comment:
```suggestion
  color: ${p => (p.priority ? p.theme.white : p.theme.gray5)};
```

Reply:
I'll get this in another PR unless there's anything else worth changing in this one, would like to merge now while the tests pass (assuming I can get review)